### ISSUE TITLE
Addressbook visibility not switchable anymore

### ISSUE THREAD

**Issue by mr-gosh** [owncloud/contacts#406]
### Steps to reproduce
1. go to contacts app
2. open the configuration gear
3. realize there is no visibility switcher in front of the addressbooks anymore
### Expected behaviour

the sivibility switcher is a great way to organize a lot of addressbooks
AND
to check if the import function is importing to the right addressbook (atm this can only be done through a third party like thunderbird or davdroid, cause the "import to" function is missing too)
### Actual behaviour

Tell us what happens instead

no toggle switch, no "import to"
### Server configuration

**Operating system**:
ubuntu 14.04 64bit

**Web server:**
apache 2.4

**Database:**
mysql

**PHP version:**
php7

**ownCloud version:** (see ownCloud admin page)
9.0.2

**Contacts version:** (see ownCloud apps page)
1.2.0.0

**Updated from an older ownCloud or fresh install:**
updates from 9.0.1

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.2.1
  - calendar: 1.1
  - comments: 0.2
  - contacts: 1.2.0.0
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_external: 0.5.2
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - music: 0.3.11
  - news: 8.7.3
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - encryption
  - external
  - notes
  - ownnote
  - tasks
  - user_external
  - user_ldap

```

**The content of config/config.php:**

```
too sensitive for such a ticket
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
local
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
### Client configuration

**Browser:**
firefox 46

**Operating system:**
ubuntu 16.04
#### Web server error log

```
gui related problem - son no error log here
```
#### ownCloud log (data/owncloud.log)

```
gui related problem - son no error log here
```
#### Browser log

``````
gui related problem - son no error log here```
``````

**Comment by skjnldsv** [comment]
@DeepDiver1975 is there an enable/disable function in core/dav?
If not, how do you think we can do this?

**Comment by mr-gosh** [comment]
just to recap:
owc8 it looked like this and you were able to select the addressbook you wanna import to:
![owc8_contacts](https://cloud.githubusercontent.com/assets/3028703/15111793/d32ff890-15e9-11e6-8cc1-7fd8712f1a62.png)

now it looks like this and it imports to the latest added addressbook:
![owc9_contacts](https://cloud.githubusercontent.com/assets/3028703/15111810/f1b351d6-15e9-11e6-9420-fba35f961299.png)

**Comment by skjnldsv** [comment]
Yes, I see exactly what you need, and I also think it was a great feature! :)
We'll see if this is possible!

**Comment by DeepDiver1975** [comment]
the calendar app uses a custom webdav property to store the visibility

**Comment by skjnldsv** [comment]
Nice! Thx!

**Comment by Jather90** [comment]
any updates on this?

**Comment by wopl** [comment]
Haven't heard anything from this. I urgently need enable/disable adressbooks feature to support multiple address books. 

As open since May I suggest, there is no interest by developers to implement this?
Any comment welcome, so that I can decide to switch platform.

Please let us know, if in scope and what estimated release date ... (hope never dies)
Wolfram

**Comment by DeepDiver1975** [comment]
> Haven't heard anything from this. I urgently need enable/disable adressbooks feature to support multiple address books.
> 
> As open since May I suggest, there is no interest by developers to implement this?
> Any comment welcome, so that I can decide to switch platform.

![this-is-open-source](https://cloud.githubusercontent.com/assets/1005065/20896299/673ca1d6-bb1d-11e6-945a-b4396afd499c.jpg)

**Comment by DeepDiver1975** [comment]
guys you are welcome to contribute - thx

**Comment by wopl** [comment]
keep calm... I am aware this is open source. I just wanted to know. Now I do. 
A bit sad, as it already was implemented in previous version.

Too many tasks already on my wash-list to open further ones.
Thx Wolfram
